pancancer-arch-3/src/test/resources/workerConfig.json

```diff
@@ -1,12 +1,10 @@
-{
-	"rabbitMQQueueName":"arch_3_worker",
-	"rabbitMQHost":"localhost",
-	"rabbitMQUser":"guest",
-	"rabbitMQPass":"guest",
-	"heartbeatRate":"2.5",
-	"preworkerSleep":"1",
-	"postworkerSleep":"1",
-	"max-runs":"1",
-	"hostUserName":"sshorser",
-	"endless":"false"
-}
\ No newline at end of file
+rabbit.rabbitMQQueueName=arch_3_worker
```

Comment:
This isn't the right format for commons config files. 
See https://github.com/CancerCollaboratory/sandbox/blob/develop/pancancer-arch-3/src/test/resources/config